extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayCreator.java

```diff
@@ -57,8 +59,17 @@ public Flyway createFlyway(DataSource dataSource) {
         configure.cleanDisabled(flywayRuntimeConfig.cleanDisabled);
         configure.baselineOnMigrate(flywayRuntimeConfig.baselineOnMigrate);
         configure.validateOnMigrate(flywayRuntimeConfig.validateOnMigrate);
-        configure.ignoreMissingMigrations(flywayRuntimeConfig.ignoreMissingMigrations);
-        configure.ignoreFutureMigrations(flywayRuntimeConfig.ignoreFutureMigrations);
+        configure.validateMigrationNaming(flywayRuntimeConfig.validateMigrationNaming);
+        List<String> patterns = new ArrayList<>(2);
+        //https://flywaydb.org/documentation/configuration/parameters/ignoreMigrationPatterns
```

Comment:
Should we implement ignoreMigrationPatterns? It can be done in a subsequent PR but it seems like something we should have.

Reply:
Sure, that would be a good enhancement to have. I'll create an issue for that